.env

```diff
@@ -0,0 +1,4 @@
+NPM_TOKEN=95ba9f82-8ec6-4a44-8209-85e4360cd194
+GH_TOKEN=0a5b546be7a5d248950fd41cf495c31b664fdd3f
```

Comment:
I don't think you meant to commit this

Reply:
Yea I had added it to my .gitignore 
escaped somehow